leadconnectorhq.com.email.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "leadconnectorhq.com",
+  "providerName": "LeadConnector",
+  "serviceId": "email",
+  "serviceName": "LeadConnector Email records",
+  "syncRedirectDomain": "domainconnect.leadconnectorhq.com,leadconnectorhq.com",
+  "version": 1,
+  "logoUrl": "https://assets.cdn.filesafe.space/knES3eSWYIsc5YSZ3YLl/media/6269a1de6e3f42c196dbd949.png",
+  "description": "Enable domain domain to work with the LeadConnector",
+  "variableDescription": "IPs: LeadConnector connection",
+  "records": [
+    {
+      "type": "MX",
+      "host": "%host%",
+      "pointsTo": "%mxValue%",
```

Comment:
I recommend signing for such open template.
Many DNS provider will reject without signing.